php/api/controller/class-audit-posts-controller.php

```diff
@@ -342,6 +344,31 @@ public function prepare_item_for_response( $post, $request ) {
 			$response->remove_link( $field );
 		}
 
+		// Only for authenticated users.
+		if ( User::authenticated() ) {
+
+			// Only interested in _audit_<standard> meta.
+			$meta = array_filter( get_post_meta( $post->ID ), function ( $k ) {
```

Comment:
The `standards` array should have the information you need. Something along the lines of:
```
// Get "standards" from the post meta.
$standards = get_post_meta( $post->ID, 'standards', true );

// Filter using the allowed standards.
if ( ! empty( $standards ) ) {
    $allowed_standards = array_keys( Audit::allowed_standards() );

    $standards = array_filter( $standards, function ( $standard ) use ( $allowed_standards ) {
        return in_array( $standard, $allowed_standards, true );
    } );
} else {
    $standards = array_keys( Audit::executable_audit_fields() );
}
```

Reply:
I have to load the meta further down anyway.  Granted the get_post_meta gets cached, but doing it here for `standards` which I am not convinced is reliable.... and doing it subsequently to populate the `_link` seems to create unnecessary queries.

What I do need to do however is to use the allowed standards filter!

Reply:
The updated item merges the old standards with the new ones here https://github.com/wptide/wp-tide-api/blob/develop/php/api/controller/class-audit-posts-controller.php#L585-L615

Reply:
The audit server should have knowledge of the standards array from the SQS message, or in the future the Firestore record. We need to normalize the way we handle the standards. Either we _use_ or _remove_ `standards` completely. We shouldn't be half in and implement multiple solutions.